packages/office-addin-sso/src/authRoute.ts

```diff
@@ -45,11 +46,20 @@ export class AuthRouter {
                         }
                     });
                     const json = await tokenResponse.json();
-
                     res.send(json);
+
+                    // Send usage data
+                    const usageDataInfo: Object = {
+                        Method: ['authRouter'],
+                        scope: [scopeName],
+                        Platform: [process.platform],
+                        Succeeded: [true]
```

Comment:
Are all of these values really supposed to be in arrays?

Reply:
this format is the expectation of the office-addin-usage-data package as it's currently implemented. I tried not doing it in an array initially and git an error when looping through the object in reportEventApplicationInsights

Reply:
OK. It looks like there are some good learnings here about how the methods exposed from the office-addin-usage-data could have been better designed.